app/representers/api/v1/question_representer.rb

```diff
@@ -23,19 +23,20 @@ class QuestionRepresenter < Roar::Decorator
              writeable: true,
              readable: true
 
-    collection :stems,
-               class: Stem,
-               decorator: StemRepresenter,
+    collection :answers,
```

Comment:
Had to put `answers` first so they will be parsed before the `stem_answers` that reference them.